File: d_rank/rank_test_problems_d_0001/java/1.java
// Paiza 問題集 Dランク・実力確認問題セット1 ダウンロード(Easy) Java編（paizaランク D 相当）
// https://paiza.jp/works/mondai/rank_test_problems_d_0001/rank_test_problems_d_0001__1
import java.util.*;

public class Main
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

        String[] vs = sc.nextLine().split(" ");
        int s = Integer.parseInt(vs[0]);
        int a = Integer.parseInt(vs[1]);
        int t = Integer.parseInt(vs[2]);

        double sa = s * a;
        double ta = t * a;
        System.out.println((int)((ta / sa) * 100.0));    
    }
}

File: d_rank/rank_test_problems_d_0001/go/1.go
// Paiza 問題集 Dランク・実力確認問題セット1 ダウンロード(Easy) Go編（paizaランク D 相当）
// https://paiza.jp/works/mondai/rank_test_problems_d_0001/rank_test_problems_d_0001__1
package main
import "fmt"

func main() {
	var s, a, t int

	fmt.Scan(&s, &a, &t)
	sa := (float32)(s * a)
	ta := (float32)(t * a)
	fmt.Println((int)((ta / sa) * 100.0))
}
